packages/client/src/components/interface/ApplicationStatusWindow.tsx

```diff
@@ -28,7 +28,7 @@ const Window = styled.div<{
   position: fixed;
   top: 64px;
   right: 0;
-  z-index: 3;
+  z-index: 1;
```

Comment:
In the case of mobile view, the header which has z-index 2 covers the status window header

![image](https://user-images.githubusercontent.com/42269993/146553467-5094d079-f33d-4a07-a643-b195ac6eaf99.png)

Reply:
![image](https://user-images.githubusercontent.com/29002716/146555845-b09df6df-57bc-4d00-8209-2c92fc1a6497.png)
Changed the z-index for smaller views.

Reply:
Thanks @Zangetsu101 , LGTM! 👍🏼